man/readtags.1.rst.in

```diff
@@ -92,6 +92,12 @@ human-readable, but when utilizing readtags output in a script or a client
 tool, ``-E`` option should be used. See ctags-client-tools(7) for more
 discussion on this.
 
+About printing input fields ({tagfile} in tags(5)) with ``-E`` option,
+the behavior of readtags is the same as ctags as explained in
+ctags-client-tools(7); the escaping rules are applied only when
```

Comment:
It's a bit weird to say readtags has the same behavior as ctags, because readtags reads and ctags writes. My suggestion:

```
About printing input fields ({tagfile} in tags(5)) with ``-E`` option: readtags
always prints the input field literally (as it is in the tags file), and when
ctags writes the tags file, the escaping rules are applied only when
``TAG_OUTPUT_MODE`` pseudo tag has "u-ctags" and ``TAG_OUTPUT_FILESEP`` has
"slash" as values for their input fields, as explained in
ctags-client-tools(7).
```

Reply:
Thank you. 

```
readtags always prints the input field literally (as it is in the tags file)
```
is not true.
I will update the description in a new commit.

Reply:
Updated.